Add field labels to tokens in "Send form results" message

shinde-rahul opened the pull request:
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | ❌
| New feature/enhancement? (use the a.x branch)      | ✔️
| Deprecations?                          | ❌
| BC breaks? (use the c.x branch)        | ❌
| Automated tests included?              | ✔️
| Related user documentation PR URL      | mautic/user-documentation#... <!-- required for new features -->
| Related developer documentation PR URL | mautic/developer-documentation-new#... <!-- required for developer-facing changes -->
| Issue(s) addressed                     | Fixes #... <!-- prefix each issue number with "Fixes #", no need to create an issue if none exists, explain below instead -->

<!--
Additionally (see https://contribute.mautic.org/contributing-to-mautic/developer/code/pull-requests#work-on-your-pull-request):
 - Always add tests and ensure they pass.
 - Bug fixes must be submitted against the lowest maintained branch where they apply
   (lowest branches are regularly merged to upper ones so they get the fixes too.)
 - Features and deprecations must be submitted against the "4.x" branch.
-->

## Description

Currently, when inserting form result tokens into the message field of the "Send form results" action, only the values are included. This makes it difficult to understand the context of each value in the resulting email. This update modifies the token insertion to include both the field label and value, improving clarity for email recipients.

### Use story
**As a** Mautic user
**I want** field labels to be included when I insert tokens into the message of the "Send form results" action
**So that** the form values in the resulting email are clear and meaningful


| Before                                 | After
| -------------------------------------- | ---
| <img width="597" alt="Screenshot 2025-05-13 at 10 29 11 AM" src="https://github.com/user-attachments/assets/3c543243-d1d2-4e2d-8fa2-f29f5e0cc691" /> |  <img width="604" alt="image" src="https://github.com/user-attachments/assets/1e55d140-ba3e-414f-ab12-8e9651c54472" />


### Changes,

- Remove unused template: `_form_submitaction_sendemail_widget.html.twig`
The template located at `app/bundles/FormBundle/Resources/views/FormTheme/SubmitAction/_form_submitaction_sendemail_widget.html.twig` is no longer used and can be safely removed. Cleaning up dead code to reduce maintenance overhead.
- Updated JavaScript code and templates to reflect the new approach for inserting form tokens.

---
### 📋 Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a form:
   * Go to the Forms section.
   * Add fields like **First Name** and **Email**.
3. Add a form action:
   * Go to the **Actions** tab.
   * Add the **"Send form results"** action.
4. Verify initial token behavior:
   * Scroll to the **Message** field in the form action.
   * Click tokens like **First Name** or **Email**.
   * Confirm they are added in the format:
     **`<label>: {formfield=field_alias}`**
     For example:
     **`First Name: {formfield=first_name}`**
5. Test adding a new field dynamically:
   * Add a new field to the form, e.g., **Last Name**.
   * Reopen the **"Send form results"** action.
6. Verify updated token insertion:
   * Scroll to the **"Click to insert field"** section.
   * Click on **"Last Name"**.
   * Confirm the following is inserted at the cursor or end of the message field:
     **`<strong>Last Name</strong>: {formfield=last_name}`**

shinde-rahul committed:
Add form field labels in result tokens inserted into editor

shinde-rahul committed:
Added acceptance test for the form

escopecz commented on app/bundles/CoreBundle/Assets/js/11.editor.js:
```diff
@@ -116,12 +116,16 @@ Mautic.configureDynamicContentAtWhoTokens = function() {
     }
 };
 
-Mautic.insertTextInEditor = function (obj, text) {
```
Please document this BC break in the `UPGRADE-7.0.md` file.

shinde-rahul replied:
Updated `UPGRADE-7.0.md` documenting the change.

escopecz reviewed (commented):
The code looks good. Can you please mention in the PR description why the codeception library needed an upgrade and why the `app/bundles/FormBundle/Resources/views/FormTheme/SubmitAction/_form_submitaction_sendemail_widget.html.twig` file was deleted?

shinde-rahul committed:
Revert composer files to avoid unintended upgrade of codeception/module-asserts

shinde-rahul committed:
Document BC break in UPGRADE-7.0.md: Removed Mautic.insertTextInEditor

Added note under JavaScript section about the removal of Mautic.insertTextInEditor. Developers should use Mautic.insertHtmlInEditor instead.

shinde-rahul commented:
@escopecz, The composer file reverted to its previous state. I ran `composer require --dev codeception/module-asserts` when Codeception failed and the editor did not provide assertion autocomplete.

escopecz reviewed (approved):
The code looks good, the enhancement works as described:

https://github.com/user-attachments/assets/11fb301d-2680-4e8d-96ed-05d326595d97
